.pre-commit-config.yaml

```diff
@@ -16,23 +16,10 @@
   - id: pretty-format-json
     exclude: .vscode/*
     args: [--autofix, --indent=2]
-- repo: https://github.com/pre-commit/mirrors-eslint
-  rev: v6.7.0
+- repo: local
   hooks:
-  - id: eslint
-    args: [--fix]
-    exclude: \..*\.js
-    additional_dependencies:
-    - babel-eslint@10.0.1
-    - eslint@6.7.0
-    - eslint-config-standard@14.1.0
-    - eslint-friendly-formatter@4.0.1
-    - eslint-loader@3.0.2
-    - eslint-plugin-chai-friendly@0.5.0
-    - eslint-plugin-cucumber@1.4.0
-    - eslint-plugin-cypress@2.6.1
-    - eslint-plugin-import@2.18.2
-    - eslint-plugin-node@10.0.0
-    - eslint-plugin-promise@4.2.1
-    - eslint-plugin-standard@4.0.1
-    - eslint-plugin-vue@6.0.1
\ No newline at end of file
+    - id: eslint
+      name: eslint
+      entry: yarn lint
+      language: system
+      pass_filenames: false
```

Comment:
That new way of checking our code's linting is much better. It uses the frontends' lint command reliably. We could add a test hook in that very same way.